src/graphqlFetch.ts

```diff
@@ -236,6 +154,102 @@ app.get("/:org/:repo", async (req, res) => {
   res.json(reponse);
 });
 
+async function getDashData(organisation: string) {
+  let querystring = "org:" + organisation + " topic:hacktoberfest";
+  let data: GraphQlQueryResponseData;
+  try {
+    data = await octo.graphql(
+      `query getDashData($organisation: String!, $querystring: String!) {
+  search(last: 100, type: REPOSITORY, query: $querystring) {
+    repos: nodes {
+      ... on Repository {
+        name
+        url
+        description
+        repositoryTopics(first: 100) {
+          nodes {
+            topic {
+              name
+            }
+          }
+        }
+        defaultBranchRef {
+          name
+          target {
+            ... on Commit {
+              history {
+                totalCount
+              }
+            }
+          }
+        }
+        openIssues: issues(states: OPEN) {
+          totalCount
+        }
+        prOpen: pullRequests(states: OPEN) {
+          totalCount
+        }
+      }
+    }
+  }
+  organization(login: $organisation) {
+    name
+    avatarUrl
+    description
+    url
+  }
+}
+      `,
+      {
+        organisation: organisation,
+        querystring: querystring,
+      }
+    );
+  } catch (error) {
+    if (error instanceof GraphqlResponseError) {
+      return error.message;
+    } else {
+      return "Server Error!";
+    }
+  }
+  let response: orgData = {
+    name: data.organization.name,
+    avatarUrl: data.organization.avatarUrl,
+    description: data.organization.description,
+    url: data.organization.url,
+    hfestRepos: data.search.repos.length,
+    repos: [],
+  };
+
+  for (let node of data.search.repos) {
+    let repository = {
+      name: node.name,
+      url: node.url,
+      description: node.description,
+      topics: node.repositoryTopics.nodes.map((nodes) => nodes.topic.name),
```

Comment:
No need to map nodes array if every node is just returning the topic name.

Reply:
The GraphQL server returns the nodes in the following way:
```"nodes": [
              {
                "topic": {
                  "name": "powershell"
                }
              },
              {
                "topic": {
                  "name": "windows"
                }
              }
               ...],
```
Thus it is imperative to map it in an array which can be returned as a regular array by our backend